docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
@takluyver, I'm in favor of keeping the pyqt4 instructions (and sorry for the name errors, I should have verified them first).

The thing is only leaving `python3-pyqt5` seems to imply that `qtconsole` only works in Python 3 and with PyQt5, which is obviously not true.